apps/dashboard/src/@/components/ui/dialog.tsx

```diff
@@ -21,7 +21,7 @@ const DialogOverlay = React.forwardRef<
   <DialogPrimitive.Overlay
     ref={ref}
     className={cn(
-      "fixed inset-0 bg-black/80  data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0",
+      "z-[10000] fixed inset-0 bg-black/80  data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0",
```

Comment:
last time we did z-index stuff like this it caused a bunch of downstream weirdness are you *sure* this is OK to do?

Reply:
removed - moved it to a specific instance